bbf49e758b353cabf698a73d6a5f3820af06e367
src/test/java/com/yelp/nrtsearch/server/grpc/LuceneServerTest.java
--- 
+++ 
@@ -240,7 +240,6 @@
         List<String> RETRIEVE = Arrays.asList("doc_id", "new_text_field");
 
         Query query = Query.newBuilder()
-                .setQueryType(QueryType.TERM_QUERY)
                 .setTermQuery(TermQuery.newBuilder()
                         .setField("new_text_field")
                         .setTextValue("updated"))
@@ -379,7 +378,6 @@
         assertEquals(2, statsResponse.getMaxDoc());
 
         Query query = Query.newBuilder()
-                .setQueryType(QueryType.TERM_QUERY)
                 .setTermQuery(TermQuery.newBuilder()
                         .setField("count")
                         .setIntValue(7))